build/index_core_compatible.js

```diff
@@ -1,20 +1,37 @@
-import Experiment from '../es6/experiment';
-import Interpreter from '../es6/interpreter';
-import RandomPlanoutCoreCompatible from '../es6/ops/randomPlanoutCoreCompatible';
-import Core from '../es6/ops/core';
-import * as Namespace from '../es6/namespace';
 import Assignment from '../es6/assignment';
+import Experiment from '../es6/experiment';
 import ExperimentSetup from '../es6/experimentSetup';
+import Interpreter from '../es6/interpreter';
+import * as LibUtils from '../es6/lib/utils';
+import Random from '../es6/ops/randomPlanoutCoreCompatible';
+import * as Base from '../es6/ops/base';
+import * as Core from '../es6/ops/core';
+import * as OpsUtils from '../es6/ops/utils';
+import * as Namespace from '../es6/namespacePlanoutCoreCompatible';
+
+OpsUtils.registerOperators({
+    "randomFloat": Random.RandomFloat,
+    "randomInteger": Random.RandomInteger,
+    "bernoulliTrial": Random.BernoulliTrial,
+    "bernoulliFilter": Random.BernoulliFilter,
+    "uniformChoice": Random.UniformChoice,
+    "weightedChoice": Random.WeightedChoice,
+    "sample": Random.Sample
+}, true);
```

Comment:
I'd like to have a working build soon (end of week would be super awesome), but I don't want to advocate for a half-baked solution. If it comes down to it, we can stay on 2.0.2 until this is ready. I think your proposal makes sense and moves closer to a dependency injection model. I'd actually considered doing something like that, but didn't know what appetite this project had for reorganization at that level. I could take a crack at it. Or if you'd prefer to own this and just knock it out, that's fine, too.